File: src/main/java/org/zenbot/timetableupdater/configuration/TimetableResourceLocationProperties.java
package org.zenbot.timetableupdater.configuration;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

import java.util.List;

@Data
@ConfigurationProperties("timetable.resources")
public class TimetableResourceLocationProperties {
    private List<String> locataions;
}

File: src/main/kotlin/org/zenbot/timetableupdater/configuration/TimetableResourceLocationProperties.kt
package org.zenbot.timetableupdater.configuration

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("timetable.resources")
class TimetableResourceLocationProperties {
    lateinit var folder: String
    lateinit var fileExtension: String
    lateinit var commentSign: String
}

